### PR TITLE
Release 20210225

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist template

- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] Correct Markdown and formatting
- [x] All content reviewed

-----
[View rendered GOVERNANCE.md](https://github.com/publiccodenet/about/blob/release-20210225/GOVERNANCE.md)
[View rendered README.md](https://github.com/publiccodenet/about/blob/release-20210225/README.md)
[View rendered activities/codebase-stewardship/product-assets-for-early-incubation.md](https://github.com/publiccodenet/about/blob/release-20210225/activities/codebase-stewardship/product-assets-for-early-incubation.md)
[View rendered activities/communication/blogging.md](https://github.com/publiccodenet/about/blob/release-20210225/activities/communication/blogging.md)
[View rendered activities/communication/get-new-illustration.md](https://github.com/publiccodenet/about/blob/release-20210225/activities/communication/get-new-illustration.md)
[View rendered activities/documentation/index.md](https://github.com/publiccodenet/about/blob/release-20210225/activities/documentation/index.md)
[View rendered activities/documentation/merge-develop-into-main.md](https://github.com/publiccodenet/about/blob/release-20210225/activities/documentation/merge-develop-into-main.md)
[View rendered contributor-guides/for-individuals.md](https://github.com/publiccodenet/about/blob/release-20210225/contributor-guides/for-individuals.md)
[View rendered contributor-guides/for-staff.md](https://github.com/publiccodenet/about/blob/release-20210225/contributor-guides/for-staff.md)
[View rendered organization/privacy.md](https://github.com/publiccodenet/about/blob/release-20210225/organization/privacy.md)

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -4,12 +4,14 @@ type: Resource
...
+This project represents the Foundation for Public Code.
+Therefore any changes adopted into the `main` branch have to be accepted by the directors of the Foundation for Public Code.
+The `sprint` branch contains all features approved by at least one director and it represents the current 'working draft' version of the Foundation.
 
 ## New content and changes into the `sprint` branch
 
 Every change and new feature can be accepted into the `develop` branch by any director that did not open that pull request.
 
-## Publishing changes from `sprint` to `master`
+## Publishing changes from `sprint` to `main`
```
Comment (bvhme) [inline GOVERNANCE.md:15]:
```suggestion
## Publishing changes from `develop` to `main`
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -4,12 +4,14 @@ type: Resource
 
 # Content governance
 
-This project represents the Foundation for Public Code. Therefore any changes adopted into the `master` branch have to be accepted by the directors of the Foundation for Public Code. The `sprint` branch contains all features approved by at least one director and it represents the current 'working draft' version of the Foundation.
+This project represents the Foundation for Public Code.
+Therefore any changes adopted into the `main` branch have to be accepted by the directors of the Foundation for Public Code.
+The `sprint` branch contains all features approved by at least one director and it represents the current 'working draft' version of the Foundation.
```
Comment (bvhme) [inline GOVERNANCE.md:9]:
```suggestion
The `develop` branch contains all features approved by at least one other committer and it represents the current 'working draft' version of the Foundation.
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -4,12 +4,14 @@ type: Resource
...
 
 ## New content and changes into the `sprint` branch
```
Comment (bvhme) [inline GOVERNANCE.md:11]:
```suggestion
## New content and changes into the `develop` branch
```

**Review by bvhme — APPROVED** [review]

**Review by ElenaFdR — COMMENTED** [review]
@bvhme, good spot on the Governance file. This looks ready to go.

**Comment by bvhme** [comment]
Are the checks done but just no checkmarks marked?

**Comment by ElenaFdR** [comment]
Reviewed and checkmarks added now!

**Review by bvhme — APPROVED** [review]

**Pull request merged**